webui/react/src/components/kit/Columns.module.scss

```diff
@@ -0,0 +1,26 @@
+.columns {
+  display: flex;
+  gap: var(--gap);
+  margin-bottom: var(--margin-bottom);
+
+  .column {
+    display: flex;
+    flex-grow: 1;
+
+    &.left {
+      justify-content: start;
+    }
+    &.center {
+      justify-content: center;
+    }
+    &.right {
+      justify-content: right;
+    }
+  }
+}
+
+@media only screen and (max-width: $breakpoint-tablet) {
```

Comment:
This causes the workspace cards to reflow as well. It's usually better to find a non media-query way to do the CSS so it's more portable, but if you want to do it with media-querys it needs to be configurable in some way.

At the most simple level we could add a `page: bool` parameter to `Columns` that indicates this is a "page-level layout". We could also do something [more complex](https://react-bootstrap.github.io/layout/grid/).